pkg/polaris/scanner.go

```diff
@@ -29,22 +29,19 @@ import (
 const (
 	polarisContainerName = "polaris"
 	configVolume         = "config"
-	polarisVersion       = "1.2"
-)
-
-var (
-	polarisContainerImage = fmt.Sprintf("quay.io/fairwinds/polaris:%s", polarisVersion)
 )
 
 type Scanner struct {
+	config    starboard.Config
 	opts      kube.ScannerOpts
 	clientset kubernetes.Interface
 	pods      *pod.Manager
 	converter Converter
 }
 
-func NewScanner(opts kube.ScannerOpts, clientset kubernetes.Interface) *Scanner {
+func NewScanner(config starboard.Config, opts kube.ScannerOpts, clientset kubernetes.Interface) *Scanner {
```

Comment:
As mentioned above and other places, Polaris should not be aware of kube-bench or kube-hunter config. Define an interface for configuring Polaris, and let starboard.ConfigData implement this interface.